lib/calendarium-anglicus/temporale/dates.rb

```diff
@@ -1,7 +1,25 @@
 module CalendariumAnglicus
   class Temporale < CalendariumRomanum::Temporale
     # dates of movable feasts
-    module Dates < CalendariumRomanum::Temporale::Dates
+    module Dates
+      WEEK = CalendariumRomanum::Temporale::WEEK
+      # TODO This piece of shameful code was written by @igneus and demonstrates
+      #   how unflexible the machinery of CalendariumRomanum::Temporale currently is.
+      #   The date computing methods are Module class methods, but as such they cannot be
+      #   conveniently inherited.
+      class << self
```

Comment:
This is literally the ugliest piece of code I've written in months :vomiting_face: 

When designing `CalendariumRomanum::Temporale::Dates` I had no idea that there is no sane way to inherit module "class methods". This is a very ugly workaround through metaprogramming.